esrally/driver/driver.py

```diff
@@ -1210,6 +1213,14 @@ def __init__(self):
         self.wakeup_interval = Worker.WAKEUP_INTERVAL_SECONDS
         self.sample_queue_size = None
 
+    @actor.no_retry("worker")  # pylint: disable=no-value-for-parameter
+    def receiveMsg_Bootstrap(self, msg, sender):
+        self.driver_actor = sender
+        # load node-specific config to have correct paths available
+        self.cfg = load_local_config(msg.config)
```

Comment:
Do we really need to attach those to `self`? If we do we should set them to `None` in `__init__` but it looks they're only used locally.

Additionally, I think having both `cfg` and `msg.config` is confusing. What do you think about `node_config` instead of `cfg`?

Reply:
Good points! I refactored these attributes in https://github.com/elastic/rally/pull/1763/commits/34566f1fc0d8ccc237af64b4a78f3ab452a1fa8b as I noticed we were needlessly setting the config twice otherwise.